django_server/.github/workflows/test.yml

```
@@ -0,0 +1,251 @@
+      - name: Run Safety check
+        run: safety check --json
+
+      - name: Run Bandit
+        run: bandit -r . -f json -o bandit-report.json
+
+      - name: Upload Security Reports
+        uses: actions/upload-artifact@v3
+        with:
+          name: security-reports
+          path: |
+            bandit-report.json
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)